src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="$(NETStandardLibraryPackage)" Version="$(NETStandardLibraryVersion)" />
+    <PackageReference Include="NETStandard.Library" Version="$(NETStandardLibraryVersion)" />
```

Comment:
@NikolaMilosavljevic @jkoritzinsky `netcoreapp.depproj` has a dependency on the 2.2 version of `NETStandard.Library` (which never shipped). I tried using the `NETStandard.Library.Ref` package (2.1.0) but that failed so I kept the `NETStandard.Library` dependency but downgraded it to `2.0.3`.

Is that change correct or do we need to reference 2.1.0?

Reply:
ping @NikolaMilosavljevic @jkoritzinsky

Reply:
This PackageReference is to the old-style netstandard package instead of the runtime pack (so NETStandard.Library). I think we should depend on the version that core-setup consumed to create NETStandard.Library.Ref 2.1.0 since Microsoft.NETCore.App supports .NET Standard 2.1, but the 2.0.3 version should work as well if we want only stable dependency versions.

Reply:
What is this even used for???  It's not for redist and it's not to preserve as a package-reference since this no longer produces a package that is used as part of the application's package graph (I believe that is what this was originally for).  Can we just delete this?

Reply:
The installer subset tests still use the package graph consumption model. They were never moved to the new model.